client/src/components/User.js

```diff
@@ -10,18 +12,28 @@ User.propTypes = {
   skills: PropTypes.arrayOf(PropTypes.string),
 }
 
-export default function User({ id, name, phone, mail, department, skills }) {
+export default function User({
+  id,
+  name,
+  phone,
+  mail,
+  department,
+  skills,
+  deleteUser,
+}) {
   return (
     <Wrapper>
+      <DeleteButtonWrapper>
+        <Button children={'X'} onClick={() => deleteUser(id)} />
+      </DeleteButtonWrapper>
       <Details>
-        <span>{id}</span>
         <span>Name: {name}</span>
```

Comment:
see my other comment. If you change to `user`, you would then have `Name: {user.name}, Phone: {user.phone}` etc here